ui/src/components/search/Search.js

```diff
@@ -279,195 +346,57 @@ const Search = ({
-                    //     },
-                    //   },
-                    //   {
-                    //     content: {
-                    //       field: 'gene_metadata.genomic_variant_count',
-                    //       displayName: '# Research Somatic Variants',
-                    //       Header: () => (
-                    //         <Row justifyContent="space-between">
-                    //           # Research Somatic Variants
-                    //           <GenomicVariantsTooltip />
-                    //         </Row>
-                    //       ),
-                    //     },
-                    //   },
-                    //   {
-                    //     content: {
-                    //       field: 'gene_metadata.clinical_variant_count',
-                    //       displayName: '# Clinical Variants',
-                    //       Header: () => (
-                    //         <Row justifyContent="space-between">
-                    //           # Clinical Variants
-                    //           <ClinicalVariantsTooltip />
-                    //         </Row>
-                    //       ),
-                    //     },
-                    //   },
-                    //   {
-                    //     content: {
-                    //       field: 'gene_metadata.histopathological_variant_count',
-                    //       displayName: '# Histo-pathological Biomarkers',
-                    //       Header: () => (
-                    //         <Row justifyContent="space-between">
-                    //           # Histo-pathological Biomarkers
-                    //           <HistopathologicalBiomarkersTooltip />
-                    //         </Row>
-                    //       ),
-                    //     },
-                    //   },
-                    // ]}
                   );
                 }}
               </SelectedModelsContext.Consumer>
             )}
           </Component>
-          <LastUpdatedDate />
         </MainCol>
       </SplitPane>
     </Col>
   );
 };
 
+// Old Table Props
+// showFilterInput={false}
+// setSelectedTableRows={(selectedRows) =>
+//   selectedModelContext?.setModels(selectedRows)
+// }
+// selectedRows={
+//   selectedModelContext?.state?.modelIds || tableContext.selectedRows
+// }
+// isLoading={
+//   savedSetsContext?.state?.loading ||
+//   tableContext.loading ||
+//   props.loading
+// }
+// onSortedChange={(sorted) => setState({ sorted })}
+// index={props.index}
+// graphqlField={props.index}
+// columnDropdownText="Columns"
+// enableSelectedTableRowsExporterFilter={true}
+// selectedRowsFilterPropertyName="_id"
+// exporterLabel="Export"
+// transformParams={(params) => ({
+//   ...params,
+//   url: `${globals.ARRANGER_API}/export/models`,
+// })}
+// fieldTypesForFilter={['text', 'keyword', 'id']}
+// enableDropDownControls={true}
+// sessionStorage={true}
+// storageKey={selectedModelContext?.storageKey}};
```

Comment:
These should be removed, I just wanted to do a final check if there's anything crucial here I've missed

Reply:
The only old props in this list that are sticking out to me are the ones related to `selectedModels`. Does the table correctly de-select models when you clear your "My Models List" now? If not, and if Arranger supports it, maybe we need to make the entire table a consumer of the SelectedModelsContext and then use the modern "selectedRows" prop equivalent? Or update the Models List to consume the Arranger context and update the selected rows.

Does Arranger still support sessionStorage as well? If so, we should re-enable those props too so users can retrieve their table state from session storage. If not, we can say it was a security concern to remove it.

I suppose these are more Arranger-specific questions, so sorry @justincorrigible but I'm gonna bug you for input again 😛